src/data/moves_info.h

```diff
@@ -16055,14 +16055,15 @@ const struct MoveInfo gMovesInfo[MOVES_COUNT_DYNAMAX] =
         .description = COMPOUND_STRING(
             "It explodes the user's head\n"
             "to damage everything around."),
-        .effect = EFFECT_MIND_BLOWN,
+        .effect = EFFECT_MAX_HP_50_RECOIL,
         .power = 150,
         .type = TYPE_FIRE,
         .accuracy = 100,
         .pp = 5,
         .target = MOVE_TARGET_FOES_AND_ALLY,
         .priority = 0,
         .category = DAMAGE_CATEGORY_SPECIAL,
+        .argument = MOVE_MIND_BLOWN,
```

Comment:
I really wish there was a better way of doing this. It's not clear to me when looking at the move data that what this does is prevent the move by Damp.

Reply:
I think a new effect is justifiable. The only reason I decided to unify them is because of tests. I didn't want to duplicate the same tests for Steel Beam because of the Damn difference.

Reply:
Hmm, I'll see if I can PR you a separation of the effects, since I feel that tests should serve the engine and not the other way around.

Reply:
What do you have in mind? Maybe I can do it.

Reply:
Oh, I guess I was able to push a commit to it ^^;